tests/test_paramviewer.py

```diff
@@ -38,7 +38,7 @@ def test_paramviewer_simple_batched(backend):
     )  # first dimension is batch dim
 
     assert pyhf.tensorlib.shape(par_slice) == (3, 3)
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[0]]) == [[1, 5, 9], [2, 6, 10]]
-    assert pyhf.tensorlib.tolist(par_slice[view.slices[1]]) == [[4, 8, 12]]
+    assert pyhf.tensorlib.tolist(par_slice[slice(1, 3)]) == [[1, 5, 9], [2, 6, 10]]
```

Comment:
is it more useful to have the transposed version? I naively expected

```
[[1, 2], [5, 6], [9, 10]]
```

to maintain batching as the first dimension since you're slicing. In this case, we're slicing _AND_ transposing.

Reply:
yeah the modifiers need it transposed

Reply:
Yeah, this is a bit of a shock to me. I would expect you adhere to the same dimension ordering as the input parameters if that's possible? It's not obvious that this would transpose...